pimpmycause/profiles/views.py

```diff
@@ -67,6 +67,9 @@ def homepage(request):
 
     return render(request, 'index.html', context)
 
+def profile_detail(request, slug):
```

Comment:
here we need to take in a request and the `user_id` the user id will come from the URL, then you can amend +    `person = PimpUser.objects.get(instance=user_id)`